roles/openafs_server/tasks/main.yaml

```diff
@@ -8,6 +8,12 @@
     - "../vars/{{ ansible_os_family }}.yaml"
     - "../vars/defaults.yaml"
 
+- name: Remove loopback address for our hostname in /etc/hosts
+  lineinfile:
+    state: absent
+    path: /etc/hosts
+    regexp: '^127\.\d+\.\d+\.\d+\s.*{{ ansible_host }}'
```

Comment:
Not quite sure you want to do exactly this.  Removing the entire localhost entry might cause grief.
You probably want to just remove the hostname and leave the "localhost" part alone..
e.g.
127.0.0.1 localhost myhost.mydomain.net  ==> 127.0.0.1 localhost

Reply:
Ah, interesting. On debian, a separate line is added,
127.0.1.1  myhost
127.0.0.1 localhost

Removing the "localhost" line would be problematic. I'll change this so we dont remove the localhost entry.